### ISSUE TITLE
Acceptance test: aws_route_table unexpected error message

### ISSUE THREAD

**Issue by gdavison** [terraform-providers/terraform-provider-aws#15274]
When an empty set of routes is passed to `aws_route_table`, it expects the following error message from the AWS API

```console
The request must contain the parameter destinationCidrBlock or destinationIpv6CidrBlock
```

This still passes in GovCloud, but the Commercial partition now supports `destinationPrefixListId`, so the error message is now

```console
The request must contain exactly one of: destinationCidrBlock, destinationIpv6CidrBlock, destinationPrefixListId
```

This should be validated in Terraform, not by the API.

### Affected Resource

* aws_route_table

**Comment by ewbankkit** [comment]
@gdavison I have fixed this in #14013.

**Comment by gdavison** [comment]
Heh, I also fixed it in #15261, @ewbankkit

**Comment by ewbankkit** [comment]
No worries.
